src/test/java/hw2/TestEx1.java

```diff
@@ -0,0 +1,127 @@
+package hw2;
+
+import org.assertj.core.api.JUnitSoftAssertions;
+import org.junit.Rule;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestEx1 extends ChromeTest{
+
+    @Test
+    public void TestsEx1(){
+        homepage();
+        login();
+        header();
+        icons();
+        iconsText();
+        frameButton();
+        leftSection();
+    }
+
+    //step 2 - Assert Browser title
+    public void homepage(){
+        assertEquals(driver.getTitle(), "Home Page");
+    }
+
+    //step 3 - Perform login
+    //step 4 - Assert Username is loggined
+    public void login(){
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+        WebElement loginname = driver.findElement(By.cssSelector("#user-name"));
+        assertEquals(loginname.getText(), "ROMAN IOVLEV");
+    }
+
+    //step 5 - Assert that there are 4 items on the header section are displayed, and they have proper texts
+
+    @Rule
+    public JUnitSoftAssertions softAssertions = new JUnitSoftAssertions();
+
+    public void header(){
+        WebElement headerHome = driver.findElement(By.linkText("Home"));
+        WebElement headerForm = driver.findElement(By.linkText("Contact form"));
+        WebElement headerService = driver.findElement(By.linkText("Service"));
+        WebElement headerMC = driver.findElement(By.linkText("Metals & Colors"));
+        List<WebElement> headers = driver.findElements(By.cssSelector("ul[class='uui-navigation nav navbar-nav m-l8']>li"));
+        int headerCount = headers.size();
+        assertThat(headerCount).isEqualTo(4);
+        softAssertions.assertThat(headerHome.isDisplayed());
+        softAssertions.assertThat(headerForm.isDisplayed());
+        softAssertions.assertThat(headerService.isDisplayed());
+        softAssertions.assertThat(headerMC.isDisplayed());
```

Comment:
Results of all assertions above are ignored. In another words you don't verify anything.

Reply:
Fix in all places in code